spyder/api/completion.py

```diff
@@ -12,182 +12,3 @@
 and workspace management functions.
```

Comment:
It seems this file can be removed now. If that's the case, please do.

Reply:
Fixed.